lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -147,6 +152,23 @@ define((require, exports, module) ->
         endRule(EQUATION_REGEX)
         endRule(LIST_REGEX)
       ]
+      "math" : [{
+            token : "comment"
+            regex : "%.*$"
+        }, {
+            token : "string.math",
+            regex : "\\${1,2}"
+            next  : "start"
+        }, {
+            token : "constant.character.escape.string.math"
+            regex : "\\\\(?:[^a-zA-Z]|[a-zA-Z]+)"
+        }, {
+            token : "error.math",
```

Comment:
it's because we should we should get out of this mode.
It is from original latex_highligt_rules.js